rwatch/ui/layer/menu_layer.c

```diff
@@ -406,7 +406,7 @@ void menu_cell_chalk_draw(GContext *ctx, const Layer *layer, const char *previou
     // Draw the selected item:
     const char *selected_title = selected->text;
     
-    bool has_icon = selected->image_res_id == NULL ? false : true;
+    bool has_icon = !selected->image_res_id ? false : true;
```

Comment:
You'll start a religious format war with things like this ;)

Reply:
It stopped the compiler from complaining. 🤷‍♂️